### PR TITLE
[EmbeddingAPI] Add embedding usecase to check UTF-8 charset diaplay w…

### PR DESCRIPTION
**Pull request by jacky-young**
…hen load a html file

-Add embedding usecase to check UTF-8 charset diaplay when load a html file
-Cover the XWalkActivity and XWalkInitializer two ways

Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: [Android]
Unit test result summary: pass 2, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-3577

### CODE REVIEW

**Comment by jacky-young** [comment]
@jennycao please help to review it.

File: usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithEncodingDisplayAsync.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Override
+    public final void onXWalkInitCompleted() {
+        setContentView(R.layout.xwview_layout);
+        StringBuffer mess = new StringBuffer();
+        mess.append("Test Purpose: \n\n")
+        .append("Verifies XWalkView can load utf-8 charset html file.\n\n")
+        .append("Expected Result:\n\n")
+        .append("Test passes if app load '你好' page.");
```
Comment (Shao-Feng) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithEncodingDisplayAsync.java]: In Java, string encoding charset is not same with file encoding charset.
In this case, how about prepare a File (utf-8 charset) under asset? if the test purpose is for validating utf-8 charset html file

Comment (jacky-young) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithEncodingDisplayAsync.java]: Before this issue is resolved, xwalkview loads string including utf-8 charset like Chinese, still gets messy code. Also I agree with your opinion, I will change to this way.

**Comment by jacky-young** [comment]
@jennycao  @Shao-Feng please help review.

**Pull request merged**